### DISCUSSION TITLE
Refresh on `backward-kill-word`

### DISCUSSION THREAD

**Discussion by cstby** [minad/vertico#442] [Q&A]
### Steps to reproduce
- `M-x find-file`
- Type `foo/bar/`
- (Files and directories inside of `foo/bar/` are listed accurately.)
- Execute `backward-kill-word` (usually bound to `C-<backspace>`)
- The prompt now shows `foo/`

### Expected result
I would expect that with `foo/` in the prompt, I would see the files and directories contained within `foo/`.

### Actual result
The list continues to display files and directories contained within `foo/bar/`.

**Comment by minad** [comment]
Hi, I cannot reproduce this issue. Vertico updates the display correctly. Maybe something in your setup is interfering? Did you try with emacs -Q? Maybe this guideline for bug reports is helpful: https://github.com/minad/consult?tab=readme-ov-file#bug-reports

**Comment by cstby** [comment]
Thanks for the above link. I was able to reproduce the issue on emacs 27.1 using `emacs -Q`. However, everything works as expected when using emacs 29.1. No further action need; I just need to upgrade to the newest version of emacs.

**Reply by minad** [reply]
Okay, thanks. Vertico should work properly on 27.1. In order to resolve this we would have to do some bisecting of the Vertico commit history.
